src/Features/DiagnosticsTestUtilities/CodeActionsLegacy/AbstractCodeActionOrUserDiagnosticTest_NoEditor.cs

```diff
@@ -405,32 +405,15 @@ protected async Task TestActionCountAsync(
     }
 
     internal Task TestInRegularAndScriptAsync(
```

Comment:
these (and other methods like these) were the core methods i wanted to remove.  They're massive, and we often take them (with small tweaks) and put in other locations.  Having these be trivial (`input, expected, optional parameters`) means getting rid of a bunch of overloads that confusing called each other (and often could do things like miss some parameter along the way).